### ISSUE TITLE
No check for empty fields or field max length is exceeded

### ISSUE THREAD

**Issue by Ririshi** [php-sepa-xml/php-sepa-xml#109]
While testing the output of the generated XML using an online checker, I found that the package will simply create the XML with whatever input is given. If a field that _must be filled_ is empty, no error is thrown. The same holds for a field that exceeds its maximum length. Perhaps it would be a good addition to add these constraints while building the output, e.g. by checking the output using a validator and throwing any errors for the user to handle.

**Comment by monofone** [comment]
Yeah, good idea, if you have any ideas on how to create validators from the XSDs post them here I'll be happy to incorporate them or as ideas for others to implement something similar.

**Comment by BorislavSabev** [comment]
Noted as a nice to have feature in a future release. 
Closing as there hasn't been any activity here.
